src/translator/quality_estimator.cpp

```diff
@@ -0,0 +1,294 @@
+#include "quality_estimator.h"
+
+namespace marian::bergamot {
+
+void UnsupervisedQualityEstimator::computeQualityScores(const Histories& histories, Response& response) const {
+  size_t sentenceIndex = 0;
+
+  for (const auto& history : histories) {
+    const std::vector<float> logProbs = std::get<1>(history->top())->tracebackWordScores();
```

Comment:
complex statement, break down and simplify

Reply:
Commit 6f68978